src/Physics/babylon.physicsImpostor.ts

```diff
@@ -638,7 +638,7 @@ module BABYLON {
                 this._tmpQuat.copyFrom(this.object.rotationQuaternion || new Quaternion());
             }
             if (!this._options.disableBidirectionalTransformation) {
-                this.object.rotationQuaternion && this._physicsEngine.getPhysicsPlugin().setPhysicsBodyTransformation(this, /*bInfo.boundingBox.centerWorld*/ this.object.getAbsolutePivotPoint(), this._tmpQuat);
+                this.object.rotationQuaternion && this._physicsEngine.getPhysicsPlugin().setPhysicsBodyTransformation(this, /*bInfo.boundingBox.centerWorld*/ this.object.getAbsolutePosition(), this._tmpQuat);

```

Comment:
@RaananW do you see any issues with this lines change?

Reply:
Rannan from slack:
I spent a lot of time checking pivot vs absolute positioning
The main issue, and the reason I chose pivot, was gltf files
Remember when they were trying to import the whole pool table? That's what actually fixed it.
And since they are identical 99% of the time (and all of the scenes I checked were working great with the pivot position), I left it this way
So I'm fine with this change